src/dlgIRC.cpp

```diff
@@ -61,265 +90,455 @@ dlgIRC::dlgIRC()
         file.close();
     }
 
-    connection->setNickName(nick);
-    mNick = nick;
-    connection->setUserName("mudlet");
-    connection->setRealName(mudlet::self()->version);
-    connection->setHost("irc.freenode.net");
-    connection->setPort(6667);
+    mNickName = nick;
+    mUserName = "mudlet";
```

Comment:
Do `mUserName = QStringLiteral("mudlet");` - it then makes it so the QString is constructed at compile-time instead of runtime. Pretty cool!

Same thing for all other places you've used raw string literals.